Chart.mqh

```diff
@@ -654,16 +654,16 @@ class Chart : public Market {
       // Nearest time.
       nBars = iBars(NULL, (ENUM_TIMEFRAMES) TimePr);
       for (i = 0; i < nBars;i++) {
-        if (iOpen(NULL,TimePr, i) >= 0.000001) {
-          if (iTime(NULL, TimePr, i) >= modeling_start_time)
+        if (::iOpen(NULL,TimePr, i) >= 0.000001) {
+          if (::iTime(NULL, TimePr, i) >= modeling_start_time)
```

Comment:
Use `Chart:: iTime` to make it work for MT5.

Reply:
Fixed